python/feldera/rest/feldera_client.py

```diff
@@ -393,6 +415,8 @@ def push_to_pipeline(
                 "json_flavor must be one of 'default', 'debezium_mysql', 'snowflake', 'kafka_connect_json_converter', 'pandas'"
             )
 
+        data = skip_none_keys_in_maps(data)

```

Comment:
this is strange. shouldn't this rather report an error?
if the data does have null keys, it is just dropped?

Reply:
Actually yeah, its better to just give a python runtime error. I'll do that.